photutils/psf/photometry.py

```diff
@@ -401,7 +404,7 @@ def do_photometry(self, image, *, mask=None, init_guesses=None):
 
         return star_groups
 
-    def nstar(self, image, star_groups, *, mask=None):
+    def nstar(self, image, star_groups, *, mask=None, progressbar=None):
```

Comment:
For consistency with with other `photutils` functions, this keyword should be called `progress_bar`.